Makefile

```diff
@@ -70,5 +65,5 @@ insights-client:
 
 
 insights-clean:
-	sudo rm -rf /etc/insights-client/* 
+	sudo rm -rf /etc/insights-client/*
```

Comment:
we should add `all` and `build` to the help as well

Reply:
Good point 👍